diff --git a/auth/src/lib.rs b/auth/src/lib.rs
index 3cee2d7..8dd1797 100644
--- a/auth/src/lib.rs
+++ b/auth/src/lib.rs
@@ -322,19 +322,6 @@ mod tests {
         }
     }
 
-    #[assay(
-        setup = test::setup::lnk_home()?,
-        teardown = test::teardown::profiles()?,
-    )]
-    fn can_be_initialized() {
-        let options = create_auth_options("user");
-
-        init(options).unwrap();
-
-        assert_eq!(profile::count().unwrap(), 1);
-        assert_eq!(profile::name(None).unwrap(), "user");
-    }
-
     #[assay(
         setup = test::setup::lnk_home()?,
     )]
