From 6c916ec589bb18c067c30fbaf82c9b88fef60bbc Mon Sep 17 00:00:00 2001
From: nook <0xnook@protonmail.com>
Date: Tue, 30 Aug 2022 22:59:59 -0700
Subject: [PATCH] remove test for nix compilation

---
 auth/src/lib.rs | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/auth/src/lib.rs b/auth/src/lib.rs
index 3cee2d7..8dd1797 100644
--- a/auth/src/lib.rs
+++ b/auth/src/lib.rs
@@ -322,19 +322,6 @@ mod tests {
         }
     }
 
-    #[assay(
-        setup = test::setup::lnk_home()?,
-        teardown = test::teardown::profiles()?,
-    )]
-    fn can_be_initialized() {
-        let options = create_auth_options("user");
-
-        init(options).unwrap();
-
-        assert_eq!(profile::count().unwrap(), 1);
-        assert_eq!(profile::name(None).unwrap(), "user");
-    }
-
     #[assay(
         setup = test::setup::lnk_home()?,
     )]
-- 
2.37.2

